### ISSUE TITLE
upload a wheel package to pypi

### ISSUE THREAD

**Issue by mgaitan** [getnikola/nikola#1434]
Wheels ( https://pypi.python.org/pypi/wheel )  are the new standard of python 
distribution and are intended to replace eggs. (see: http://pythonwheels.com/ )

I'm not sure if it could be declared as a universal wheel.

**Comment by e2jk** [comment]
Based on information from the [cheeseshop](https://pypi.python.org/pypi/wheel), it looks like upstream for Wheel is the same as upstream for pip itself, in other words the Python Packaging Authority.
The reference implementation is at https://bitbucket.org/pypa/wheel, I guess you can report this issue [there](https://bitbucket.org/pypa/wheel/issues/new).

**Comment by Kwpolska** [comment]
The manpage generation was a complete TRAINWRECK.  I removed the useless code, as the real work was done by `help2man` in `set_version.py` anyway.

We now generate 100% correct wheels.  I uploaded a wheel (generated by `./setup.py bdist_wheel`) of v7.3.0+8bcba85 (the manpage will differ *slightly* between the two, but the code stays the same)

Wheels are now added to the official release process. `*.pyc`/`__pycache__` files are to be removed before generation.  (cc @ralsina)

Wheels are also now considered the “official” distribution, as `pip` will default to them.
